package.json

```diff
@@ -35,7 +34,8 @@
     "v-hotkey3": "^0.1.4",
     "vue": "^3.1.0",
     "vue-docgen-api": "^4.43.2",
-    "write-file": "^1.0.0"
+    "write-file": "^1.0.0",
+    "mapbox-gl": "^2.15.0"
```

Comment:
Did you hit any issues with `mapbox-gl` v3? I noticed that initially you intended to use it

Reply:
Yes. With v3 version  there is an error when mapbox trigger any ajax. 
It tries to get self.location.href but hits a Proxy object and fails. It happens to any v3-minor version too.
I will open an issue to fix it later.